Server/Controls/RapidPictureBox.cs

```diff
@@ -92,8 +92,6 @@ public void UnsetFrameUpdatedEvent(FrameUpdatedEventHandler e)
         /// </summary>
         public void Start()
         {
-            _frameCounter = new FrameCounter();
```

Comment:
Well, okay. I can live with that. But the FPS won't be accurate for the first few seconds when stopping and starting remote desktop again.^^